DOC-1965: The editor displayed a notification error when it failed to retrieve a blob image uri. (#2681)
tinymce/tinymce-docs: modules/ROOT/pages/6.4.2-release-notes.adoc

// {productname} 6.4.2 also includes the following bug fixes:

=== The editor displayed a notification error when it failed to retrieve a blob image uri.

> In previous versions of {productname}, when the editor contained an image with an invalid or unreachable blob url, it would display an error notification banner within the editor viewport which would impact on the user experience.
> 
> A fix was applied in {productname} 6.4.2, which saw the removal of the error notification banner as a broken image would be visibly apparent to the user.

Reviewer: ```suggestion
In previous versions of {productname}, when the editor contained an image with an invalid or unreachable blob url, it displayed an error notification banner within the editor viewport which impacted the user experience.

For {productname} 6.4.2 the error notification banner has been removed: the visible broken image icon is notification enough for the end-user.
```

Minor edits.

> A fix was applied in {productname} 6.4.2, which saw the removal of the error notification banner as a broken image would be visiblly apparent to the user.

Reviewer 2: ```suggestion
A fix was applied in {productname} 6.4.2, which saw the removal of the error notification banner as a broken image would be visibly apparent to the user.
```
